tests/e2e/Services/Projects/ProjectsConsoleClientTest.php

```diff
@@ -3923,10 +3923,14 @@ public function testCreateProjectVariable(array $data)
             'x-appwrite-mode' => 'admin',
         ], $this->getHeaders()), [
             'key' => 'APP_TEST',
-            'value' => 'TESTINGVALUE'
+            'value' => 'TESTINGVALUE',
+            'secret' => false
```

Comment:
Let's copy improvements to function tests into console tests too - to try to make secret variable, ensure it is not returned, even in get.